libs/core/src/git.ts

```diff
@@ -41,11 +42,15 @@ export function getMergeBase({
 
 export function getDiff({ base, cwd }: BaseGitActionArgs): string {
   try {
-    return execSync(`git diff ${base} --unified=0`, {
-      maxBuffer: TEN_MEGABYTES,
-      cwd,
-      stdio: 'pipe',
-    })
+    const resolvedPath = cwd ? resolve(cwd) : cwd;
```

Comment:
I think we always pass cwd..
Actually why the cwd of execSync is not enough?

Reply:
before changes, with the original test fixtures (`cwd = libs/core/src/__fixtures__/git`), it was executing the command `git diff main --unified=0 --relative -- libs/core/src/__fixtures__/git` from the folder `libs/core/src/__fixtures__/git`.

but there is no `libs/core/src/__fixtures__/git` inside `libs/core/src/__fixtures__/git` so we have to pass to the git command an absolute path. that's why I used `path.resolve(cwd)`.

I added that ternary because `cwd` can be undefined and `path.resolve()` doesn't accept undefined args.

I could make that cleaner by executing the original `git diff` command if no cwd is provided and executing the new one if `cwd` is not undefined.